AdminServer/setup.py

```diff
@@ -17,7 +17,7 @@
   'mock',
 ]
 if sys.version_info < (3,):
-  install_requires.append('futures')
+  install_requires.extend(['futures', 'monotonic'])
```

Comment:
don't we need to install the dependency when using python 3 in order to use `import monotonic`?

Reply:
Oops, you're right. Python 3 has the monotonic function under the `time` module, so I was thinking that we don't need an extra dependency there. I guess until we fully upgrade to Python 3 it makes sense to install the monotonic module in both cases.